site-structure.json

```diff
@@ -5,6 +5,16 @@
         "sections":
         [
             {
+                "sectionName": "API",
```

Comment:
I'm not convinced that this API section should be the first thing people see in the nav bar. It seems to me that the more introductory, explanatory pages should remain at the top.